Tests/VoiceAttackPluginTests.cs

```diff
@@ -24,27 +24,62 @@ public void WriteToLog(string msg, string color = null)
 
         public void SetText(string varName, string value)
         {
-            vaVars.Add(varName, value);
+            if (vaVars.ContainsKey(varName))
```

Comment:
Are we sure this is necessary? Dictionaries are highly optimized and this looks like paying the cost for the `ContainsKey()` twice: once by us and once by the dictionary. 
I'd go with just `vaVars[varName] = value;` unless I had strong evidence that this is  big win.

Reply:
Agreed.